src/canvas/TextEditingManager.ts

```diff
@@ -13,7 +13,8 @@ export class TextEditingManager {
 
   constructor(canvas: Canvas) {
     const cb = () => {
-      const { hiddenTextarea } = canvas.getActiveObject() as ITextBehavior;
+      const { hiddenTextarea } =
+        (canvas.getActiveObject() as ITextBehavior) || {};
       hiddenTextarea && hiddenTextarea.focus();
```

Comment:
Would it make sense to replace the two lines with this? Or do we foresee using grabbing more properties in the future?
```suggestion
      (canvas.getActiveObject() as ITextBehavior)?.hiddenTextarea?.focus();
```

Reply:
@asturur doesn't like how it transpiles.
I prefer it/don't care about transpiling

Reply:
If it makes a minor performance diff, I get it. Good to know.

Reply:
We might change a build tool and/or target more recent browsers making these considerations redundant

Reply:
is way larger more than a performance diff. Here the performance is not meaningful.

it becomes something like this:

```
(_canvas$getActiveObje = canvas.getActiveObject()) === null || _canvas$getActiveObje === void 0 || (_canvas$getActiveObje = _canvas$getActiveObje.hiddenTextarea) === null || _canvas$getActiveObje === void 0 ? void 0 : _canvas$getActiveObje.focus();
```

I can't really like it.